interface/themes/style_tan.css

```diff
@@ -7,9 +7,8 @@
 body {
     margin: 0px 0px 0px 2px;
     /* set the base font and size for all DOM children */
-    font-family: 'FontAwesome' !important;
+    font-family: 'open sans', 'source_sans_pro','FontAwesome'!important; 
```

Comment:
just so you know.
In OpenEMR 5.0.1, can choose font and font size in globals:
http://www.open-emr.org/wiki/index.php/Administration_Globals#Default_font_.28need_to_logout.2Flogin_after_change_this_setting.29